EVEStandard/API/Fittings.cs

```diff
@@ -60,15 +60,15 @@ public async Task<ESIModelDTO<List<CharacterFitting>>> GetFittingsV1Async(AuthDT
         /// <param name="auth">The <see cref="AuthDTO"/> object.</param>
         /// <param name="fitting">Details about the new fitting.</param>
         /// <returns><see cref="ESIModelDTO{T}"/> containing a list of fittings.</returns>
-        public async Task<ESIModelDTO<long>> CreateFittingV1Async(AuthDTO auth, ShipFitting fitting)
+        public async Task<ESIModelDTO<ShipFittingCreated>> CreateFittingV2Async(AuthDTO auth, ShipFitting fitting)
```

Comment:
Is what is returned something more beyond a long now? If not, why the model?

Reply:
Ah, I see. Its an object being returned instead of just a number.

Reply:
Yes, alternatively we could unwrap the object in the method and return 'long', but I think I'd rather return exactly what the API returns (in case it was to return any additional properties in the future).